web/packages/teleport/src/Discover/Shared/ResourceKind.ts

```diff
@@ -56,5 +59,7 @@ export function resourceKindToPreferredResource(
       return ClusterResource.RESOURCE_KUBERNETES;
     case ResourceKind.Server:
       return ClusterResource.RESOURCE_SERVER_SSH;
+    case ResourceKind.Discovery:
```

Comment:
@michellescripts 

what's the use case of this function?
Are we going to break anything in user prefs?

Reply:
From what I saw in the code it shouldn't affect anything for 2 reasons at once - this is used in sorting resource by preferred choice, but `RESOURCE_UNSPECIFIED` will not be present in preferred user choice and discovery resource probably will not be present in the list of resources to choose from anyway. I've added it just in case and to be consistent. But I'll wait to see what @michellescripts will say 👍

Reply:
Looking this over, I agree with @AntonAM on both cases. This should have no negative impact on user prefs/sorting. Thanks for the ping @tigrato

Reply:
Thank you!